### ISSUE TITLE
[enterprise-3.10] Edit suggested in file getting_started/install_openshift.adoc

### ISSUE THREAD

**Issue by whitingjr** [openshift/openshift-docs#11815]
### Which section(s) is the issue in?

Run the Installation Playbooks

### What needs fixing?

A new step needs adding after step 2. To instruct the user to set up htpasswd configuration.
Otherwise step 4 will fail.

**Comment by kalexand-rh** [comment]
@whitingjr, is that always true, though? I thought you had to uncomment lines to activate htpasswd.

**Comment by whitingjr** [comment]
I have come across an issue where the cluster install fails. When the htpasswd file is not present.
 I've been told to do changes so the file can be found. Isn't that what you suggested @vrutkovs ?

**Comment by vrutkovs** [comment]
So there's three options on how to configure htpasswd auth:

* Set `openshift_master_identity_providers` only
   This would assume `/etc/origin/master/htpasswd` is already configured and present on the host
* Set `openshift_master_htpasswd_file`
   This option would copy a local htpasswd file to the host
* Set `openshift_master_htpasswd_users`
   This var would be used to generate a new htpasswd file on the host

Seems our docs don't mention additional options and expect user to create htpasswd manually, so it needs to be extended

**Comment by DanyC97** [comment]
@whitingjr would you mind extending your PR to include the options provided by @vrutkovs ? (thanks for the very clear info !)

**Comment by whitingjr** [comment]
@DanyC97 Sure. Where exactly do you want @vrutkovs options. In the section [Making configuration changes using Ansible](https://docs.openshift.com/container-platform/3.10/install_config/master_node_configuration.html#master-node-config-ansible) or [Run the Installation Playbooks](https://docs.openshift.com/container-platform/3.10/getting_started/install_openshift.html#run-the-installation-playbooks) ?

**Comment by vrutkovs** [comment]
This configuration is optional, imho it should go to master-node config docs

**Comment by whitingjr** [comment]
Ok the issue turned out to not needing a fix. Making a change to the documentation unnecessary.

 I think this issue is no longer needed. I am happy this issue be closed.

**Comment by vrutkovs** [comment]
These options and difference still needs to be documented though

**Comment by whitingjr** [comment]
Ok I will do that.

**Comment by whitingjr** [comment]
Changes to the docs have been pushed.
 I've taken into consideration what's been [discussed](https://bugzilla.redhat.com/show_bug.cgi?id=1565447#c14) as the options (or not) for configuration.

**Comment by kalexand-rh** [comment]
@whitingjr, I don't see an update to https://github.com/openshift/openshift-docs/pull/11817. Did you put them in a different PR?

**Comment by whitingjr** [comment]
@kalexand-rh don't think so. I pushed another commit to the [branch](https://github.com/openshift/openshift-docs/pull/11817/commits). You should see it with this hash 84d7c5c9ed72e8ddee6c1a66185dea974d6b2608.

**Comment by whitingjr** [comment]
@kalexand-rh are you wanting an edit similar to  @vrutkovs put in his comment ?

**Comment by kalexand-rh** [comment]
@whitingjr, I think that's what I was trying to find in your update.

**Comment by whitingjr** [comment]
@kalexand-rh I did nothing nearly like what you were looking for. I'll go back and do it right this time.

**Comment by whitingjr** [comment]
@kalexand-rh a change has been pushed to the branch.
